src/components/search/index.js

```diff
@@ -188,7 +165,7 @@ function SearchContent({
         {...{ collapse, focus }}
       />
       {searchQuery?.length > 0 && focus && (
-        <div className="hits-wrapper fixed w-full grid z-10 left-0 right-0 overflow-hidden bg-white border border-gray-500 md:absolute md:rounded-md scrolling-touch">
+        <div className="hits-wrapper fixed w-full grid z-10 left-0 right-0 overflow-hidden bg-white border border-gray-500 top-0 md:absolute md:rounded-md scrolling-touch transform translate-y-10 md:translate-y-12">
```

Comment:
can we put the `top-0` alongside `left-0 right-0` ?

Reply:
@bloudermilk Do we have a guideline in which order we should display the class names?

Reply:
@loiclefloch we don't have guidelines here but it would be good to think about, if we can create/load a jslint rule to enforce it. In the meantime I agree in this specific case if @chenxsan doesn't mind making this small change.

Reply:
yep - i wonder if prettier or other folks have this... seems super simple and should just be auto-formatted